tox.ini

```diff
@@ -1,7 +1,10 @@
 #
 # Configuration for tox and pytest
 
+# first five ignores are defaults per http://flake8.readthedocs.io/en/latest/config.html#default
 [flake8]
+ignore = E123, E133, E226, E241, E242, W503, E402
+max-line-length = 77
 exclude = dist,docs,*.egg-info,.git,ref,_scratch,.tox
```

Comment:
@eupharis - I was just looking at the features/steps/test_files/hdr-header-props.pptx. It looks like you embedded the header in a table. Is there some reason for that? I was just thinking we're probably better off for initial testing purposes if it's just as simple as possible, like the 'S1P1' in a single paragraph. Was there a specific reason you wanted to use a table?

Also, the document part is named document2.xml, which seems a little odd. How did you create the .docx file?

Also, what did S1P1 stand for? I'm thinking section 1, paragraph 1, which seems like a sensible convention except we'd need it to be S1HP1 or something, wouldn't we? To indicate whether it's a header or footer?